src/js/components/modals/InstallPackageModal.js

```diff
@@ -89,19 +92,29 @@ class InstallPackageModal
         currentTab: "defaultInstall",
         truncatedPreInstallNotes: true
       });
+
+      return;
     }
-    // If Opening
-    if (!props.open && nextProps.open) {
-      // If default - auto-install
-      if (!nextProps.advancedConfig) {
-        this.handleInstallPackage();
-      } else {
-        // Show advanced install
-        this.setState({
-          currentTab: "advancedInstall"
-        });
-      }
+
+    if (nextProps.isBetaPackage) {
```

Comment:
Thinking about this, I think we might be over-engineering here. If any of the props change, we actually want to fall into this code block again to force a reset in beta-terms related state.